### PR TITLE
Add responsive attribute to Sortable AMP ad slots.

### PR DESCRIPTION
**Pull request by SpLouk**
When data-repsonsive = 'true', this slot will be treated by Sortable
as a responsive ad unit.

### CODE REVIEW

File: ads/sortable.js [hunk 1]
```diff
@@ -25,9 +25,10 @@ export function sortable(global, data) {
```
Comment (Rooke) [inline ads/sortable.js]: You can put in `data-responsive` as an optional parameter in the above line.

File: ads/sortable.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ __Required:__
...
 `type` - always set to "sortable"
+
+__Optional:__
+
+`data-reponsive` - when set to true indicates that the ad slot has multiple potential sizes.
  
 
-No explicit configuration is needed for a given sortable amp-ad, though each site must be set up beforehand with [Sortable](http://sortable.com). The site name `ampproject.org` can be used for testing. Note that only the two examples above will show an ad properly.
\ No newline at end of file
+No explicit configuration is needed for a given sortable amp-ad, though each site must be set up beforehand with [Sortable](http://sortable.com). The site name `ampproject.org` can be used for testing. Note that only the two examples above will show an ad properly.
```
Comment (Rooke) [inline ads/sortable.md]: Not sure where the diff is here. Best to reduce noise for the sake of outside developers/PRs..

**Comment by Rooke** [comment]
Looks good. Squash it, then we can do a pull request from your branch.

File: ads/sortable.md (hunk 2 above)
Comment (SpLouk) [inline ads/sortable.md]: I think my editor added a newline at the end of the file

**Comment by SpLouk** [comment]
Rebased.

**Comment by Rooke** [comment]
Apologies for dragging this out. Do you think it's wise to add `renderStartImplemented: true` to our config in this PR?

**Comment by SpLouk** [comment]
Well... I actually took out the call to that function in another PR, since it seems to be optional. I can add it back in, but I'm not really sure that it adds value for us.

**Comment by Rooke** [comment]
It might do nothing now, but doesn't it add future flexibility? e.g. automatic declaration of the resize request.

**Comment by SpLouk** [comment]
Good call.

**Comment by SpLouk** [comment]
we will have to merge [this](https://github.com/sortable/tng/pull/4739) before this gets merged into the amp project.

File: ads/sortable.js [hunk 3]
```diff
@@ -21,13 +21,15 @@ import {loadScript, validateData} from '../3p/3p';
  * @param {!Object} data
  */
 export function sortable(global, data) {
-  validateData(data, ['site', 'name']);
+  validateData(data, ['site', 'name'], ['responsive']);
 
   const slot = global.document.getElementById('c');
   const ad = global.document.createElement('div');
+  const size = (data.responsive === 'true') ? 'auto'
```
Comment (Rooke) [inline ads/sortable.js]: nitpick: have both conditions on a new line, i.e.:
``` javascript
const size = (data.responsive === 'true') ? 
     'auto'
    : data.width + 'x' + data.height;
```

**Pull request closed without merging**